### ISSUE TITLE
Explain arbitrary classification model

### ISSUE THREAD

**Issue by rdelbel** [TeamHG-Memex/eli5#246]
The lime python package is able to explain an arbitrary classificaiton model as long as "... the classifier implements a function that takes in raw text or a numpy array and outputs a probability for each class."

There is some reference that potentially this is possible in eli5 
"TextExplainer allows to explain predictions of any text classifier using LIME algorithm (Ribeiro et al., 2016). There are utilities for using LIME with non-text data and arbitrary black-box classifiers as well, but this feature is currently experimental."

I am wondering how it is possible to do this with eli5.

**Comment by lopuhin** [comment]
@rdelbel for text inputs there is a whole tutorial here: http://eli5.readthedocs.io/en/latest/tutorials/black-box-text-classifiers.html and eli5 supports this case well. There is currently no support for image inputs. Here https://github.com/TeamHG-Memex/eli5/blob/master/notebooks/LIME%20and%20synthetic%20data.ipynb you can see a notebook that applies eli5.lime to arbitrary numberical inputs, but here support is much less complete than would be needed to make it robust and easy to use.
